mofa/model.py

```diff
@@ -148,6 +148,17 @@ class LigandDescription:
     def atoms(self):
         return read_from_string(self.xyz, "xyz")
 
+    def find_anchor_C_idx(self) -> list:
+        """Finds the indices of carbon atoms in the anchors
+
+        Returns:
+            list of indices
+        """
+        xyzdf = pd.read_csv(StringIO(self.xyz), sep=r"\s+", header=None, index_col=None, names=["el", "x", "y", "z"], skiprows=2)
```

Comment:
Not a correctness, but a style point: would you mind switching to using ASE rather than Pandas? ASE has a rich set of options for dealing with atomic coordinates and parsing files which you end up re-creating with Pandas.